### PR TITLE
add full size tests somatic

### PR DESCRIPTION
**Pull request by ggabernet**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_full_somatic.config [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+params {
+    config_profile_name        = 'Full test profile for somatic VC'
+    config_profile_description = 'Full test dataset to check somatic VC pipeline function'
+
+    // Input data for full size test
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/sarek/testdata/csv/HCC1395_WXS_somatic_full_test.csv'
+
+    // Other params
+    tools = 'strelka,mutect2,deepvariant,freebayes,ascat,manta,vep'
```
Comment (FriederikeHanssen) [inline conf/test_full_somatic.config:21]:
```suggestion
    tools = 'strelka,mutect2,freebayes,ascat,manta,vep'
```

**Review by maxulysse — APPROVED** [review]

File: conf/test_full_somatic.config (hunk 1 above)
Comment (FriederikeHanssen) [reply conf/test_full_somatic.config:21]: should we test all possible tools here? So also msisensorpro, cnvkit and controlfreec?

**Comment by FriederikeHanssen** [comment]
also should we add --split_fastq for some speed up?

**Comment by ggabernet** [comment]
> also should we add --split_fastq for some speed up?

yes, sure, which setting would you recommend, just `--split_fastq`?

File: conf/test_full_somatic.config [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    genome = 'GATK.GRCh38'
+    vep_cache_version = 105
```
[the remark this answers is not in the document]
Comment (maxulysse) [reply conf/test_full_somatic.config:23]: you shouldn't need to provide it anyway, you're using iGenomes, it's correctly stated and managed there, it fails when specified as a params.
not added to ignored params yet, will do that soon

File: .github/workflows/awsfulltest.yml [hunk 3]
```diff
@@ -20,9 +20,27 @@ jobs:
           workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
           access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
           compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
-          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/sarek/work-${{ github.sha }}
+          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/sarek/work-${{ github.sha }}/germline_test
```
Comment (apeltzer) [inline .github/workflows/awsfulltest.yml:23]: Somatic tests, but germline output folder? Sure this shouldn't read somatic?

File: conf/test_full_somatic.config (hunk 1 above)
Comment (FriederikeHanssen) [reply conf/test_full_somatic.config]: should hopefully display correctly now https://nf-co.re/sarek/dev

**Pull request merged**